docs/DeveloperGuide.md

```diff
@@ -1,25 +1,196 @@
 # Developer Guide
 
-## Design & implementation
+## Design 
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+##Implementation
+For public users, the list of commands is shown below:
 
+* Display selected store sample menu: `menu`
+* Display all reviews of the selected store : `reviews`
+* Add a new review of the selected store: `add`
+* Goes back to home page to select canteen: `home`
+* Display all the stores of the selected canteen: `list`
+* Exiting the application: `exit`
+
+
+For admin, the list of commands is shown below:
+
+ 1. View canteens
+ 2. Add canteen
+ 3. Add store in canteen
+ 4. Delete canteen
+ 5. Delete store in canteen
+ 6. Delete reviews
+ 7. Exit
+
+
+<!-- NOT TOO SURE HOW THIS PART FITS IN
+
+#### Sequence Diagram for `admin`
+When the user enters `2` to go into the admin page, the AdminVerification() is called. It will ask the user to input 
+the password. Then it will check the input against the set password. If fails then the user have to enter again or enter
+`exit` to exit the application.
+--> 
+
+
+### Main NusFoodReviews
+
+#### Implementation
+![Main Sequence Diagram](./img/Main.png)
+
+When the application is launched, a Ui object and Parser object is instantiated.
+To instantiate the Parser object, the main NusFoodReviews and Ui object is passed.
+
+### [Public user] Display Menus Feature
+#### Implementation
+![DisplayMenus Sequence Diagram](./img/DisplayMenus.png)
```

Comment:
It would be better if you could have an example of the input and output for the command.